play/src/main/scala/org/lyranthe/prometheus/client/integration/play/filters/PrometheusFilter.scala

```diff
@@ -0,0 +1,65 @@
+package org.lyranthe.prometheus.client.integration.play.filters
+
+import com.google.inject.{Inject, Singleton}
+import org.lyranthe.prometheus.client._
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+@Singleton
+class PrometheusFilter @Inject()()(implicit val registry: Registry,
+                                   executionContext: ExecutionContext)
```

Comment:
I've reconsidered - given how quick the use of the EC should be, I think it's okay to use the global execution context for it.